includes/admin/load.php

```diff
@@ -204,51 +210,75 @@ function perflab_dismiss_wp_pointer_wrapper() {
  * Callback function to handle admin scripts.
  *
  * @since 2.8.0
+ * @since n.e.x.t Renamed to perflab_enqueue_features_page_scripts().
  */
-function perflab_enqueue_modules_page_scripts() {
-	wp_enqueue_script( 'updates' );
-
-	wp_localize_script(
-		'updates',
-		'_wpUpdatesItemCounts',
-		array(
-			'settings' => array(
-				'totals' => wp_get_update_data(),
-			),
-		)
-	);
-
+function perflab_enqueue_features_page_scripts() {
 	wp_enqueue_script( 'thickbox' );
 	wp_enqueue_style( 'thickbox' );
+}
 
-	wp_enqueue_script( 'plugin-install' );
-
-	wp_enqueue_script(
-		'perflab-plugin-management',
-		plugin_dir_url( __FILE__ ) . 'js/perflab-plugin-management.js',
-		array( 'jquery' ),
-		'1.0.0',
-		array(
-			'in_footer' => true,
-			'strategy'  => 'defer',
-		)
-	);
+/**
+ * Register admin actions for handling performance feature plugin installation/activation.
+ *
+ * @since n.e.x.t
+ */
+function perflab_register_admin_actions() {
+	add_action( 'admin_action_perflab_install_activate_plugins', 'perflab_install_activate_plugins_callback' );
 }
+add_action( 'admin_init', 'perflab_register_admin_actions' );
```

Comment:
Why the extra function? Can't we just do the following?

```suggestion
add_action( 'admin_action_perflab_install_activate_plugins', 'perflab_install_activate_plugins_callback' );
```

Reply:
Updated!